components/server/src/prebuilds/github-app-rules.ts

```diff
@@ -34,13 +34,23 @@ export class GithubAppRules {
         return deepmerge(defaultConfig, cfg.github);
     }
 
+    /**
+     *
+     * @deprecated
+     */
```

Comment:
Would it make sense to move this logic to `checkPrebuildPrecondition`?

Reply:
basically same as https://github.com/gitpod-io/gitpod/pull/18716#discussion_r1332994036